tests/test_peak_time_range.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 31 10:04:56 2023
+
+@author: claudia
+"""
+
+import os
+import pytest
+import numpy as np
+import random
+import math
+from scipy.stats import norm
+
+from ramp.core.core import User, Appliance
+from ramp.core.core import UseCase
+from ramp.core.initialise import initialise_inputs
+from ramp.core.stochastic_process import calc_peak_time_range
+
+def load_usecase(j=None, fname=None):
+    peak_enlarge, user_list, num_profiles = initialise_inputs(
+        j, fname, num_profiles=1
+    )
+    return user_list
+
+class TestPeakTime:
+    def test_calc_peak_time_range(self):
+            j=1
+            peak_enlarge=0.15
+            user_list=load_usecase(j,peak_enlarge )
```

Comment:
```suggestion
            user_list=load_usecase(j)
```
Here `load_usecase` should only take one argument